.teamcity/src/subprojects/release/publishing/PublishMavenBuild.kt

```diff
@@ -0,0 +1,29 @@
+package subprojects.release.publishing
+
+import jetbrains.buildServer.configs.kotlin.v10.*
+import jetbrains.buildServer.configs.kotlin.v2019_2.BuildType
+import subprojects.build.core.*
+import java.lang.RuntimeException
+
+class PublishMavenBuild(private val build: Build) : BuildType({
+    id("KtorPublishMavenBuild_${build.name}".toExtId())
+    name = "Publish ${build.name} to Maven"
+
+    dependencies {
+        val buildId = build.build?.id
```

Comment:
`build.build` looks weird

Reply:
Suggestions? I don't see what I should rename the variable that would be different to the property name